client/client.go

```diff
@@ -234,6 +242,21 @@ func NewClientWithAddress(address string) (client Client, err error) {
 	return newClientWithConnectionAndCancelFunc(conn, ctxCancel), nil
 }
 
+func getClientTimeoutSeconds() (int, error) {
+	timeoutStr := os.Getenv(clientTimoutSecondsEnvVarName)
+	if len(timeoutStr) == 0 {
+		return clientDefaultTimoutSeconds, nil
+	}
+	timeoutVar, err := strconv.Atoi(timeoutStr)
+	if err != nil {
+		return 0, err
+	}
+	if timeoutVar <= 0 {
+		timeoutVar = clientDefaultTimoutSeconds
+	}
```

Comment:
This behavior seems a bit inconsistent ... If the env var is set incorrectly then we are erroring out ... but when a negative value is set (incorrect value) then we are defaulting to 5 seconds. IMO these two paths should have the same behavior, either error out in both cases or default in both cases and print a warning log. 
@dapr/maintainers-go-sdk WDYT?

Reply:
Modified, incorrect value is also erroring out now.